src/collections/Breadcrumb/BreadcrumbDivider.js

```diff
@@ -0,0 +1,55 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import META from '../../utils/Meta'
+import { customPropTypes } from '../../utils/propUtils'
+import Icon from 'src/elements/Icon/Icon'
+
+/**
+ * A divider sub-component for Breadcrumb component.
+ */
+function BreadcrumbDivider(props) {
+  const {
+    children, icon, className, ...rest,
+  } = props
+
+  const classes = cx(
+    className,
+    'divider',
+  )
+
+  if (icon) {
+    return <Icon className={classes} name={icon} {...rest} />
```

Comment:
If you'd like to wait on this PR, let's leave the Icon with the name prop.  If you'd like to get this one merged before continuing on with the others, then just use `className` here and the Icon PR can update it.

I'll leave the allocation up to you.